Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -458,17 +468,22 @@ def set_name_and_icon(body: dict, method: str):
             body['icon_url'] = BOT_ICON_URL
 
 
-def send_slack_request_sync(client: slack_sdk.WebClient, method: str, http_verb: str = 'POST', file_: str = '',
-                            body: dict = None) -> SlackResponse:
+def send_slack_request_sync(
+    client: slack_sdk.WebClient,
+    method: str = '',
+    http_verb: str = 'POST',
+    body: Optional[dict] = None,
+    file_upload_params: Optional[FileUploadParams] = None,
+) -> SlackResponse:
     """
     Sends a request to slack API while handling rate limit errors.
 
     Args:
         client: The slack client.
         method: The method to use.
         http_verb: The HTTP method to use.
-        file_: A file path to send.
         body: The request body.
+        file_upload_params: An instance of FileUploadParams.
```

Comment:
Please add to the docstring that this argument is related only to the file upload command.